src/data/adventures.txt

```diff
@@ -705,3 +705,23 @@ PirateRealm Island	adventure=531	DiffLevel: mid Env: outdoor Stat: 0 nowander	St
 PirateRealm Island	adventure=531	DiffLevel: mid Env: outdoor Stat: 0 nowander	Red Roger's Fortress
 PirateRealm Island	adventure=531	DiffLevel: mid Env: outdoor Stat: 0 nowander	Jack's Hideout
 PirateRealm Island	adventure=531	DiffLevel: mid Env: outdoor Stat: 0 nowander	The Temple
+
+# Obselete areas that nevertheless are reported as possibilities for the autumnaton
```

Comment:
```suggestion
# Obsolete areas that nevertheless are reported as possibilities for the autumnaton
```

Reply:
As long as you are doing that, move this line from farther up in the file
```
# Woods	adventure=99	DiffLevel: unknown Env: outdoor	The Road to White Citadel
```
and do the same thing to it.